tests/plugins/autocomplete/manual/a11ydivarea.html

```diff
@@ -5,6 +5,11 @@
 </div>
 
 <script>
+	// (4712).
```

Comment:
Please add `#` before issue reference.
```suggestion
	// (#4712).
```